core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -16,10 +16,18 @@
  */
 package kafka.tools;
 
+import java.io.IOException;
```

Comment:
I have done re-ordering to follow java standard imports first and then later project specific imports.
Please let me know if you want me to revert it back